README.md

```diff
@@ -35,51 +37,18 @@ The ingress host name is follow, where `<base_hostname>` is your deployment's ba
 * Installation of the Ansible collections on the control node.
   * If installing from the Ansible Automation Hub, then run `ansible-galaxy install redhat.trusted_profile_analyzer`.
   * If installing from this Git repository, then clone it locally, and run `ansible-galaxy collection install -r requirements.yml`.
-* An OpenID Connect (OIDC) provider, such as [Keycloak](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/sso/).
-* A PostgreSQL instance
-* SQS like [Kafka](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/amq_streams/)
-* S3 service or S3 compatible service
-* Optional:
-  Installation of the `podman` binaries to verify that the RHTPA service is working as expected.
+    
+The following external services are provided by the customers.
+* An OpenID Connect (OIDC) provider, such [RedHat Single Sign On](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/sso/) or AWS Cognito.
```

Comment:
On the first use of an acronym, we should spell it out.

`... or AWS Cognito.` --> `... or Amazon Web Services (AWS) Cognito.`

Now you can start using AWS elsewhere in the doc.

Reply:
Suggestion applied